gravitee-am-ui/src/app/utils/time-range-utils.ts

```diff
@@ -45,29 +45,29 @@ export const availableTimeRanges: TimeRange[] = [
   {
     id: '1d',
     name: 'Today',
-    value: 1,
-    unit: 'days',
+    value: 24,
+    unit: 'hours',
     interval: 1000 * 60 * 60,
   },
   {
     id: '7d',
     name: 'This week',
-    value: 1,
-    unit: 'weeks',
+    value: 7,
+    unit: 'days',
     interval: 1000 * 60 * 60 * 24,
   },
   {
     id: '30d',
     name: 'This month',
-    value: 1,
-    unit: 'months',
-    interval: 1000 * 60 * 60 * 24,
+    value: 4,
+    unit: 'weeks',
+    interval: 1000 * 60 * 60 * 24 * 7,
   },
   {
     id: '90d',
     name: 'Last 90 days',
     value: 3,
     unit: 'months',
-    interval: 1000 * 60 * 60 * 24,
+    interval: 1000 * 60 * 60 * 24 * 30,
```

Comment:
**thought**: I'm wondering if we will have enough interval to make the graph useful 🤔

Reply:
Thought the same. It may not return enough info for the graph to be useful

Reply:
what do you propose?

Reply:
For a 3 month period, maybe a one week interval could be enough but a one month interval is not enough IMO. You should ask to product team.